File: src/main/java/net/alexherman/example/ExampleConsumer.java
package net.alexherman.example;

import net.alexherman.example.api.DefaultApi;

public class ExampleConsumer {

    private final DefaultApi swansonQuotesApi;

    public static ExampleConsumer create() {
        return new ExampleConsumer(new DefaultApi());
    }

    public ExampleConsumer(DefaultApi swansonQuotesApi) {
        this.swansonQuotesApi = swansonQuotesApi;
    }

    public String dropKnowledge() {
        return swansonQuotesApi.quotesGet().get(0);
    }
}

File: lib/example_consumer.rb
require "swanson_quotes_client"

class ExampleConsumer

    def self.create
        ExampleConsumer.new(SwansonQuotesClient::DefaultApi.new())
    end

    def initialize(default_api)
        @default_api = default_api;
    end

    def dropKnowledge
        @default_api.quotes_get()[0]
    end

end
